portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,30 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.gson.Gson;
+import java.util.*;
 
 /** Servlet that returns a hardcoded message */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  ArrayList<String> commentsList = new ArrayList(Arrays.asList("hello", "world"));
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Hello User!");
+    // Convert ArrayList to Json
+    Gson gson = new Gson();
+    String json = gson.toJson(commentsList);
+    // Send JSON as response
```

Comment:
nit: it's probably ok to omit this comment since it's pretty clear what the code does